source/scss/components/_teaser.scss

```diff
@@ -95,15 +103,6 @@
   margin-bottom: 1rem;
 }
 
-// TODO From old Teaser button. Should this be removed?
```

Comment:
Are you sure, this is not needed for the normal Content-Teasers anymore?

Reply:
This is a good point and that is why I commented it before, just to be sure, it does affect or not affect something else.
So, the teaser-cta is a block-container, that does not have any definitions, other than 
```
.teaser-cta {
  display: flex;
  justify-content: flex-end;
  margin-top: auto;
}
```
In the same file. (At some point I'll move this code into its own file, because you can find it inside the _teaser.scss, but it is its own component.)

### My first question was: What would put the button to the bottom of the teaser?
It is ` margin-top: auto;`.

### The second question is: What would put the button to the right side?
It is `justify-content: flex-end;`

The .teaser-cta block-element (`display: flex;` is an extension of `display:block;`) automatically has a width of 100% per default.
<img width="212" alt="Screenshot 2019-08-23 at 10 17 34" src="https://user-images.githubusercontent.com/4340405/63577748-66f77780-c58f-11e9-9f5e-376000e89f37.png">

We can add: `justify-content: flex-end;`to move its child elements to the right side:
<img width="218" alt="Screenshot 2019-08-23 at 10 16 15" src="https://user-images.githubusercontent.com/4340405/63577784-7aa2de00-c58f-11e9-98c4-0de9176758a0.png">
As long as we don't want to have anything next to it, this is the best way.

Another possibilty would be to use `align-items: flex-end;` where the parent element needs to be defined as flex-element, which moves the child-flex-element to the right side:
<img width="212" alt="Screenshot 2019-08-23 at 10 16 28" src="https://user-images.githubusercontent.com/4340405/63577918-b5a51180-c58f-11e9-9767-a967b13b5451.png">

### Next question would be: Why is there an a selector inside the .teaser-cta?

One thing is, that our content teaser, does not have an anchor tag:
<img width="242" alt="Screenshot 2019-08-23 at 10 24 44" src="https://user-images.githubusercontent.com/4340405/63578233-4f6cbe80-c590-11e9-83a2-84dec154cc02.png">
And that's also the case for the link-price component:
<img width="428" alt="Screenshot 2019-08-23 at 10 26 12" src="https://user-images.githubusercontent.com/4340405/63578332-804cf380-c590-11e9-9248-dd346f3f6076.png">

The anchor-tag of the elements are placed and act as containers. And we have learned, that we can not have anchors inside anchors – Browsers forbid it.

Unfortunately, but actually fortunately, CSS can not tell us, when a selector can not find a DOM element. That is why we can not detect code, that does not do anything anymore.

### Last but not least: The ellipsis.
It does not apply, because:

1. the content of the buttons should be fully visible, therefore the editors find a wording, that fits into the buttons
2. it does not apply technically, since the browser integrated ellipsis, as well as our own cross-browser ellipsis need a well defined width and height to work with. The content would simply break to the next line, if it was wide enough.
3. again, since there is no anchor anymore, it does not apply at all.